src/main/java/seedu/exercise/logic/commands/RedoCommand.java

```diff
@@ -41,11 +44,14 @@ public CommandResult execute(Model model) throws CommandException {
         //TODO Refactor this out for v1.4!!!!!!!!!!!!!!!!!! I want to puke looking at this code.
```

Comment:
Don't forget the TODO. is there a better way to do this please. The code here is truly abominable

Reply:
resolved in latest